readers/cassandra/README.md

```diff
@@ -13,6 +13,9 @@ default values.
 | MF_CASSANDRA_READER_PORT        | Service HTTP port                              | 8180           |
 | MF_CASSANDRA_READER_DB_CLUSTER  | Cassandra cluster comma separated addresses    | 127.0.0.1      |
 | MF_CASSANDRA_READER_DB_KEYSPACE | Cassandra keyspace name                        | mainflux       |
+| MF_CASSANDRA_READER_DB_USERNAME | Cassandra DB username                          |                |
```

Comment:
I would put an empty string here and for all empty variables

Reply:
Maybe, but then we have to be consistent and add `"` for all values, which is too much for this PR, should be separate. Also, this will be presented as a table, so I think that it's clear that it's empty value

Reply:
I should remove it from Postgres reader and writer then